frontend/src/metabase/nav/components/ProfileLink.jsx

```diff
@@ -87,68 +82,77 @@ export default class ProfileLink extends Component {
     ].filter(Boolean);
   };
 
-  render() {
-    const { modalOpen } = this.state;
-    const { tag, date, ...versionExtra } = MetabaseSettings.get("version");
-    // don't show trademark if application name is whitelabeled
-    const showTrademark = t`Metabase` === "Metabase";
-    return (
-      <div>
-        <EntityMenu
-          tooltip={t`Settings`}
-          items={this.generateOptionsForUser()}
-          triggerIcon="gear"
-          triggerProps={{
-            color: color("text-medium"),
-            hover: {
-              backgroundColor: color("brand"),
-              color: color("text-white"),
-            },
-          }}
-        />
-        {modalOpen === "about" ? (
-          <Modal small onClose={this.closeModal}>
-            <div className="px4 pt4 pb2 text-centered relative">
-              <div className="text-brand pb2">
-                <LogoIcon height={48} />
-              </div>
-              <h2
-                style={{ fontSize: "1.75em" }}
-                className="text-dark"
-              >{t`Thanks for using Metabase!`}</h2>
-              <div className="pt2">
-                <h3 className="text-dark mb1">
-                  {t`You're on version`} {tag}
-                </h3>
-                <p className="text-medium text-bold">
-                  {t`Built on`} {date}
-                </p>
-                {!/^v\d+\.\d+\.\d+$/.test(tag) && (
-                  <div>
-                    {_.map(versionExtra, (value, key) => (
-                      <p key={key} className="text-medium text-bold">
-                        {capitalize(key)}: {value}
-                      </p>
-                    ))}
-                  </div>
-                )}
-              </div>
+  useEffect(() => {
+    const isAdmin = user.is_superuser;
+    if (isAdmin && MetabaseSettings.isPaidPlan()) {
+      UtilApi.bug_report_details().then(setBugReportDetails);
+    }
+  }, [user.is_superuser]);
```

Comment:
Instead of adding a `componentDidMount`, I think this is a perfect opportunity to convert this class component to a functional component as it isn't too complex.